GraphRag/requirements.txt

```diff
@@ -0,0 +1,6 @@
+llama-index-embeddings-huggingface
+llama-index-llms-ollama
+llama-index
+pyvis
+tree-sitter==0.21.3
+tree-sitter-languages
```

Comment:
Where are we using tree-sitter package in the code?

Reply:
When we are using a code splitter it uses the same internally

Reply:
Then the code splitter should be installing it internally you should not be needed to provide it as a dependency for your own module.

Reply:
when I was trying in collab file there was a bug appearing which got solved only after installing `tree-sitter==0.21.3`
`tree-sitter-languages` they weren't installed internally that's why I added them to make code bug free

Reply:
Okay